Externals/Logitech/Include/LogitechLEDLib.h

```diff
@@ -0,0 +1,171 @@
+//
+// Logitech Gaming LED SDK
+//
+// Copyright (C) 2011-2014 Logitech. All rights reserved.
```

Comment:
GPL conflicts with the "nontransferable" part of that EULA. It also requires the right to modify the SDK (and have source provided to do so)

Still, better than the sdk to integrate valve's steam controller.